app/controllers/concerns/invitation_controller_concerns.rb

```diff
@@ -38,6 +37,7 @@ def accept
     end
 
     def reject
+      @workshop = WorkshopPresenter.decorate(@invitation.parent)
```

Comment:
@despo any reason why we are not using `@invitation.workshop` here?

Reply:
I *think* the idea when InvitationControllerConcerns was first implemented was for it to be reused amongst different types of events, but that no longer seems to be the case as the only other model that uses if (CourseInvitation) is no longer used in the app, so I will remove the use of `parent` here.

see here:. https://github.com/codebar/planner/blob/5bc08f09196b91b210c1899205424d2d5a791768/app/models 
https://github.com/codebar/planner/blob/master/app/models/workshop_invitation.rb#L30/course_invitation.rb#L9